src/components/case_manager/AllStudents.tsx

```diff
@@ -5,10 +5,16 @@ import styles from "../../styles/Dashboard.module.css";
 import PersonCreationForm from "./PersonCreationForm";
 
 const AllStudentsPage = () => {
+  const { data: me } = trpc.getMe.useQuery(); //adds user_id
+  const user_id = me?.user_id || ""; //temp solution due to typing errors
```

Comment:
we should bail early in `handleSubmit()` if `!me` instead of providing this default, since the mutation will fail if `assigned_case_manager_id = ""`

Reply:
I think about removing lines 8 and 9 entirely and using `const { userId } = req.ctx.auth;` in the backend.